api/openapi.yaml

```diff
@@ -955,7 +956,7 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: "#/components/schemas/LedgerGrantResponse"
+                  $ref: "#/components/schemas/LedgerGrantWithLedgerIDResponse"
```

Comment:
This name is a bit weird. When do you return a response without an ID?

Reply:
Fair, the 2 list endpoints had different types prior (i think), changed both to use the same